app/src/main/java/com/mealsmadeeasy/ui/home/suggestions/SuggestionsAdapter.kt

```diff
@@ -6,12 +6,17 @@ import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
-import android.widget.Toast
 import com.mealsmadeeasy.R
 import com.mealsmadeeasy.model.Meal
 import com.squareup.picasso.Picasso
 
-class SuggestionsAdapter(meals: List<Meal> = emptyList()) : RecyclerView.Adapter<SuggestionViewHolder>() {
+private lateinit var suggestionListener : SuggestionClickListener
+
+class SuggestionsAdapter(meals: List<Meal> = emptyList(), suggestionClickListener: SuggestionClickListener) : RecyclerView.Adapter<SuggestionViewHolder>() {
```

Comment:
Let's use the function syntax here. Can we also put the arguments on their own line? It's getting a bit crowded here.